src/main/java/com/couchbase/lite/DatabaseUpgrade.java

```diff
@@ -83,15 +83,10 @@ protected boolean importData() {
             }
 
             // Open destination database:
-            boolean isOpen = false;
-            Throwable error = null;
             try {
-                isOpen = db.open();
+                db.open();
             } catch (CouchbaseLiteException e) {
-                error = e;
-            }
-            if (!isOpen) {
-                Log.e(TAG, "Upgrade failed:  Couldn't open new db: %s", db.toString(), error);
+                Log.e(TAG, "Upgrade failed:  Couldn't open new db: %s", db.toString(), e);
```

Comment:
Why don't just put e in the 3rd parameter? Anyway I think this is a bit off topic of this PR.